src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -511,10 +532,74 @@ void nl_Chip_DeviceController_SetLogFilter(uint8_t category)
 #endif
 }
 
+CHIP_ERROR nl_Chip_DeviceController_Connect(chip::DeviceController::ChipDeviceController * devCtrl, BLE_CONNECTION_OBJECT connObj,
+                                            uint32_t setupPINCode, OnConnectFunct onConnect, OnMessageFunct onMessage,
+                                            OnErrorFunct onError)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    chip::RendezvousParameters params =
+        chip::RendezvousParameters().SetSetupPINCode(setupPINCode).SetConnectionObject(connObj).SetBleLayer(&sBle);
+    ;
+    err = devCtrl->ConnectDevice(kRemoteDeviceId, params, (void *) devCtrl, onConnect, onMessage, onError);
+    SuccessOrExit(err);
```

Comment:
I find this just plain noisy: extra err definition, success or exit, goto label etc. 

How about:

```
chip::RendezvousParameters params = chip::RendezvousParameters()
        .SetSetupPINCode(setupPINCode)
        .SetConnectionObject(connObj)
        .SetBleLayer(&sBle);

return devCtrl->ConnectDevice(kRemoteDeviceId, params, (void *) devCtrl, onConnect, onMessage, onError);